<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>View2</class>
 <widget class="QDialog" name="View2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>View2</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>421</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>X axis</string>
   </property>
   <widget class="QSlider" name="fixedX">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="value">
     <number>500</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="maxX">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>1000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="minX">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="minXLabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="maxXLabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="allX">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>70</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>All</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>421</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Y axis</string>
   </property>
   <widget class="QSlider" name="fixedY">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="value">
     <number>500</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="maxY">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="value">
     <number>1000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="minY">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="minYLabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="maxYLabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="allY">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>70</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>All</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>421</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Z axis</string>
   </property>
   <widget class="QSlider" name="fixedZ">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="value">
     <number>500</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="maxZ">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="value">
     <number>1000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="minZ">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="minZLabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="maxZLabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="allZ">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>70</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>All</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>421</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Rendering Orientation</string>
   </property>
   <widget class="QCheckBox" name="xyPlane">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>130</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>XY plane</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="xzPlane">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>130</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>XZ plane</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="yzPlane">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>130</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>YZ plane</string>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="wholePlane1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>261</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>YZ plane</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="wholePlane2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>321</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>XZ plane</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="wholePlane3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>261</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>XY plane</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>xyPlane</tabstop>
  <tabstop>xzPlane</tabstop>
  <tabstop>yzPlane</tabstop>
  <tabstop>wholePlane1</tabstop>
  <tabstop>minX</tabstop>
  <tabstop>maxX</tabstop>
  <tabstop>fixedX</tabstop>
  <tabstop>allX</tabstop>
  <tabstop>wholePlane2</tabstop>
  <tabstop>minY</tabstop>
  <tabstop>maxY</tabstop>
  <tabstop>fixedY</tabstop>
  <tabstop>allY</tabstop>
  <tabstop>wholePlane3</tabstop>
  <tabstop>minZ</tabstop>
  <tabstop>maxZ</tabstop>
  <tabstop>fixedZ</tabstop>
  <tabstop>allZ</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
